jetbrains-core/resources/telemetryOverride.json

```diff
@@ -10,6 +10,12 @@
             "name": "experimentId",
             "type": "string",
             "description": "The id of the experiment being activated or deactivated"
+        },
+        {
+            "name": "toolId",
+            "type": "string",
+            "description": "The tool being installed",
+            "allowedValues": ["ssm-plugin", "sam-cli"]
```

Comment:
nit: `session-manager-plugin` and `aws-sam-cli`?

Reply:
+1